Context/GroupableContextInterface.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Context;
+
+/**
+ * Context with groups.
+ *
+ * @author Ener-Getick <egetick@gmail.com>
+ */
+interface GroupableContextInterface extends ContextInterface
```

Comment:
I'm wondering if we should not remove the sub-context interfaces and focus on the most used features.